src/runtime_src/tools/scripts/petalinux.sh

```diff
@@ -152,7 +152,11 @@ petalinux-build
 cd $ORIGINAL_DIR
 echo " * Copying PetaLinux boot files (from: $PWD)"
 cp ./${PETALINUX_NAME}/images/linux/image.ub ${XRT_REPO_DIR}/src/platform/zcu102ng/src/a53/xrt/image/image.ub
-mkdir -p ${XRT_REPO_DIR}/src/platform/zcu102ng/src/boot
+
+if [ ! -d ${XRT_REPO_DIR}/src/platform/zcu102ng/src/boot ] ; then
+  mkdir -p ${XRT_REPO_DIR}/src/platform/zcu102ng/src/boot
+fi
+
```

Comment:
This change is not necessary. "mkdir -p" will return without error if directory existed.